scripts/docker/zeppelin/bin/Dockerfile

```diff
@@ -63,12 +63,11 @@ RUN echo "$LOG_TAG Install python related packages" && \
     apt-get install -y gfortran && \
     # numerical/algebra packages
     apt-get install -y libblas-dev libatlas-dev liblapack-dev && \
-    # font, image for matplotlib
+    # font, image
     apt-get install -y libpng-dev libfreetype6-dev libxft-dev && \
     # for tkinter
     apt-get install -y python-tk libxml2-dev libxslt-dev zlib1g-dev && \
-    pip install numpy && \
-    pip install matplotlib
+    pip install numpy==1.12.1 pandas==0.21.1 matplotlib==2.1.1 pandasql==0.7.3 ipython==5.4.1 jupyter_client==5.1.0 ipykernel==4.7.0 bokeh==0.12.10 ggplot==0.11.5 grpcio==1.8.2 bkzep==0.4.0
```

Comment:
This library list came from https://github.com/apache/zeppelin/blob/master/testing/install_external_dependencies.sh#L48-L49